### DISCUSSION TITLE
Run with Expo

### DISCUSSION THREAD

**Discussion by SebasScript** [mybigday/whisper.rn#23] [Q&A]
Wow, this is super existing. Just to make sure I am getting this right, this runs the interference on the device?
Would you need a fast smart phone like iphone, samsung galaxy to run this?
And any guides on running this with Expo?

**Comment by jhen0409** [comment]
It currently runs on CPUs, and you can get better performance on iOS devices (even on an iPod Touch 7, it is acceptable to me), and it also utilizes the Accelerate framework. It may even support CoreML in the future. (See https://github.com/ggerganov/whisper.cpp/pull/566)

However, it might not be ideal on Android, as I haven't done much testing on it, so I won't say too much.

For Expo project it required to prebuild (Use `expo prebuild` or`expo run:ios`), I might update it in readme. Also, you can see the related issue #13.

**Reply by UchennaOkafor** [reply]
@jhen0409 

Is it possible for this to be made compatible without having to do prebuild? Can it work with expo-dev-client?
